### ISSUE TITLE
Custom Permissions Plugin?

### ISSUE THREAD

**Issue by deepsarda** [discordeno/cache-proxy#12]
The need for a custom permission plugin arises as the current permission plugin is incompatible to this plugin.

**Comment by itohatweb** [comment]
Maybe it will be compatible in the future 🤔🤫🤯

**Comment by Skillz4Killz** [comment]
Good point, but i think we should try to make them both similar api(both being the two cache plugins) so permissionca can work with both

**Comment by deepsarda** [comment]
> Good point, but i think we should try to make them both similar api(both being the two cache plugins) so permissionca can work with both

Getting data from the database is async so i dont think its possible
As getting from the old cache is sync
